.travis.yml

```diff
@@ -19,3 +21,9 @@ before_script:
 
 script:
   - npm test
+
+matrix:
+  allow_failures:
```

Comment:
So, in fxa-profile-server, I didn't put allow_failures in place since I want to know if there is a failure. With allowed_failures, there is no notification. (If there was a failure and no fix were immediately available, then I would use allowed_failures). My two cents.